bazel_tools/nixpkgs-disable-http2.patch

```diff
@@ -11,12 +11,3 @@ index 2f2c83d..61123c6 100644
      if repository_ctx.attr.nix_file and repository_ctx.attr.nix_file_content:
          fail("Specify one of 'nix_file' or 'nix_file_content', but not both.")
      elif repository_ctx.attr.nix_file:
-@@ -186,6 +186,8 @@ def _nixpkgs_package_impl(repository_ctx):
-         "bazel-support/nix-out-link",
-     ])
- 
-+    expr_args.extend(["--option", "http2", "false"])
```

Comment:
Why is this unnecessary now?

Reply:
It's duplicated further up in [line 10](https://github.com/digital-asset/daml/blob/d16a4b3b3cbdbf9f820585467ee594409b0d41b6/bazel_tools/nixpkgs-disable-http2.patch#L10). So, the flag is still there.